File: cs/proxy/hall/FriendRedPacketOperation.cs
/// <summary>
/// 
/// </summary>
namespace org.zgl.service.server.hall{
	[Proxy(gameId = -10)]
	public interface FriendRedPacketOperation : ITcpAsyncService { 
		/// <summary>
		/// 发好友红包
		/// </summary>
		void giveRedEnvelopes(long targetUid,long gold);
		/// <summary>
		/// 发好友红包 ----> <<giveRedEnvelopes();回调>>
		/// </summary>
		void giveRedEnvelopes2CallBack(RedPacketDto callBackParam);
		/// <summary>
		/// 领好友红包
		/// </summary>
		void bonus(int id);
		/// <summary>
		/// 领好友红包 ----> <<bonus();回调>>
		/// </summary>
		void bonus2CallBack(RedPacketDto callBackParam);
	}
}

File: componet/src/main/java/org/zgl/service/server/hall/FriendRedPacketOperation.java
package org.zgl.service.server.hall;

import org.zgl.dto.clinet.hall.RedPacketDto;
import org.zgl.proxy.MethodDesc;
import org.zgl.proxy.rule.ITcpAsyncService;

/**
 * @author： 猪哥亮
 * @创建时间： 2018/8/18
 * @文件描述：
 */
public interface FriendRedPacketOperation extends ITcpAsyncService {
    @MethodDesc("发好友红包")
    RedPacketDto giveRedEnvelopes(long targetUid, long gold);
    @MethodDesc("领好友红包")
    RedPacketDto bonus(int id);
}
